src/pages/About.js

```diff
@@ -1,13 +1,128 @@
-import React from 'react';
-import '../style/main.scss';
-
-function About() {
-    return (
-        <div>
-            <h1>About</h1>
-            <img src="https://cdn.dribbble.com/users/1355613/screenshots/6141198/fishing.jpg" alt="img" />
-        </div>
-    );
-}
+import React from 'react'
+import '../style/main.scss'
+import { Composition } from 'atomic-layout'
+import { createGlobalStyle } from 'styled-components'
+import { Box } from 'atomic-layout'
```

Comment:
You also have an import from `atomic-layout` on the line 3. Please combine them:

```diff
- import { Composition } from 'atomic-layout'
- import { Box } from 'atomic-layout'
+ import { Box, Composition } from 'atomic-layout'
```